src/game/game.c

```diff
@@ -182,6 +183,10 @@ void game_run(void)
 void game_draw(void)
 {
     window_draw(0);
+    if (game_state_is_paused()) {
+        sound_city_set_volume(0);
+        sound_speech_set_volume(0);
+    }
```

Comment:
Regardless of actually agreeing or disagreeing to the proposed change, this code will turn off the volume until you manually restore it, even after you unpause the game, so you can remove it.

Reply:
I have tested, and the sounds came back after unpausing.